tests/plugin_tests/json_rpc.cpp

```diff
@@ -87,16 +87,16 @@ BOOST_AUTO_TEST_CASE( basic_validation )
       make_request( request, -32600 );
 
       request = "{\"jsonrpc\": \"2.0\", \"method\": \"xyz\", \"params\": [], \"id\": 1}";
-      make_request( request, -32601 );
+      make_request( request, -32002 );
```

Comment:
To prevent this type of change in the future, we should move the error code defs to the jsonrpc plugin header.